zubhub_frontend/zubhub/src/assets/css/index.css

```diff
@@ -596,6 +596,9 @@ html,
   display: none !important;
 }
 
-.MuiInputBase-root #phone{
+.MuiInputBase-root #phone {
   width: 80%;
-}
\ No newline at end of file
+}
+.MuiGrid-root #Numcenter {
```

Comment:
Classes are better in this case,
Instead of line height we could use flexbox and align items center.

Try to avoid `!important` its gonna cause problems

Reply:
but without `!important` the styles are not reflecting

Reply:
i will modify the changes

Reply:
There might be styles overriding because of specificity or using !important.

if its specificity you can change the css selector so this styles has more preference,

if its because of !important, we should fix the root problem instead of keep adding !important

Reply:
by using classes and line height changes are happening but with flex there are unwanted changes in the form 
<img width="615" alt="Screenshot 2023-03-09 at 7 27 56 PM" src="https://user-images.githubusercontent.com/88829894/224048596-9ff3ed22-cf4f-488b-b4b2-69709f0c4474.png">

Reply:
Can i proceed the changes in the pr

Reply:
Can you push your changes? It shouldn’t effect other things when you change the circle to flexbox

Reply:
i have pushed the changes can you kindly review it ?